apps/chat/package.json

```diff
@@ -16,8 +16,9 @@
     "@aws-amplify/core": "^4.3.12",
     "@aws-amplify/storage": "^4.4.13",
     "amazon-chime-sdk-component-library-react": "^3.2.0",
+    "amazon-chime-sdk-js": "^3.6.0",
```

Comment:
Sichao had removed the JS SDK dependency as component library includes it as a dependency, any reason to add this?

Reply:
Needed a higher version/latest release hence why added it.  Is there a better way to do that?

Reply:
I think its fine just wanted to check cause I updated the component library to latest JS SDK but we have not released it yet.